tests/sealing/host/host.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/tests.h>
+#include <openenclave/internal/error.h>
+#include "sealing_u.h"
+
+static oe_enclave_t *_create_enclave(const char *path)
+{
+    static const uint32_t flags = oe_get_create_flags();
+    oe_enclave_t *enclave;
+    oe_result_t result;
+
+    result = oe_create_sealing_enclave(path, OE_ENCLAVE_TYPE_SGX, flags,
```

Comment:
Why OE_ENCLAVE_TYPE_SGX and not OE_ENCLAVE_TYPE_AUTO?
I don't see any reason this code should be inherently SGX specific.

Reply:
Good catch. Will change.